scripts/local/docker-compose.yml

```diff
@@ -0,0 +1,148 @@
+# This is the docker-compose script for local development.
+# For the latest version of the docker images see:
```

Comment:
what does "latest" mean in this context?  It could mean the tag latest (which is commonly used) and lead to confusion as we don't use a "latest tag" - we only use testing and the official build release versions for tags.

Reply:
Not sure, the error message you get if you don't have the "testing" tag says it's looking for "latest":

ERROR: manifest for azureiotpcs/iothub-manager-java:latest not found

 but when I put in the "latest" tag it doesn't work. I'm guessing there's somewhere in the code that the error message is saying it's looking for latest but really it's looking for testing.